sqlglot/dialects/hive.py

```diff
@@ -109,6 +109,10 @@ def _array_sort_sql(self: generator.Generator, expression: exp.ArraySort) -> str
     return f"SORT_ARRAY({self.sql(expression, 'this')})"
 
 
+def notnullcolumnconstraint_sql(self, expression: exp.NotNullColumnConstraint) -> str:
```

Comment:
move this directly into the generator, remove the lambda that pass in

Reply:
Done